examples/record-type-inclusion/record_type_inclusion.description

```diff
@@ -0,0 +1,6 @@
+// Record type inclusion provides a convenient way to copy the fields defined in a record type descriptor
```

Comment:
```suggestion
// Record-type inclusion provides a convenient way to copy the fields defined in a record-type descriptor
```

Reply:
Not a hyphenated word

Reply:
It should be hyphenated since it is an adjective for the noun `descriptor`.

Reply:
The term is "type descriptor". A record type descriptor is one type of type descriptors

Reply:
I'm not sure if it is an adjective. It's rather a kind of descriptor. The spec uses "type descriptor" and other languages also seem to use the same.